src/plugins/telemetry/include/plugins/telemetry/telemetry.h

```diff
@@ -122,6 +122,17 @@ class Telemetry : public PluginBase {
         float yaw_rad_s; /**< @brief Yaw angular velocity */
     };
 
+    /**
+     * @brief Ground speed type.
+     *
+     * Metrics typically displayed on a HUD for fixed wing aircraft.
+     */
+    struct VfrHUD {
+        float airspeed; /**< @brief Current indicated airspeed (IAS) in metres/second. */
```

Comment:
```suggestion
        float airspeed_m_s; /**< @brief Current indicated airspeed (IAS) in metres/second. */
```

We add the units usually, look at the other ones in this file for the example 🙂